app/src/sharedTest/java/org/oppia/android/app/devoptions/DeveloperOptionsFragmentTest.kt

```diff
@@ -0,0 +1,524 @@
+        itemPosition = 0,
+        targetViewId = R.id.mark_stories_completed_text_view,
+        stringIdToMatch = R.string.developer_options_mark_stories_completed
+      )
+      verifyTextOnDeveloperOptionsListItemAtPosition(
+        itemPosition = 0,
+        targetViewId = R.id.mark_topics_completed_text_view,
+        stringIdToMatch = R.string.developer_options_mark_topics_completed
+      )
+    }
+  }
+
+  @Test
+  fun testDeveloperOptionsFragment_viewLogsIsDisplayed() {
+    launch<DeveloperOptionsTestActivity>(
+      createDeveloperOptionsTestActivityIntent(internalProfileId)
+    ).use {
+      testCoroutineDispatchers.runCurrent()
+      scrollToPosition(position = 1)
+      verifyItemDisplayedOnDeveloperOptionsListItem(
+        itemPosition = 1,
+        targetView = R.id.view_logs_text_view
+      )
+      verifyTextOnDeveloperOptionsListItemAtPosition(
+        itemPosition = 1,
+        targetViewId = R.id.event_logs_text_view,
+        stringIdToMatch = R.string.developer_options_event_logs
+      )
+    }
+  }
+
+  @Test
+  fun testDeveloperOptionsFragment_configChange_viewLogsIsDisplayed() {
+    launch<DeveloperOptionsTestActivity>(
+      createDeveloperOptionsTestActivityIntent(internalProfileId)
+    ).use {
+      testCoroutineDispatchers.runCurrent()
+      onView(isRoot()).perform(orientationLandscape())
+      scrollToPosition(position = 1)
+      verifyItemDisplayedOnDeveloperOptionsListItem(
+        itemPosition = 1,
+        targetView = R.id.view_logs_text_view
+      )
+      verifyTextOnDeveloperOptionsListItemAtPosition(
+        itemPosition = 1,
+        targetViewId = R.id.event_logs_text_view,
+        stringIdToMatch = R.string.developer_options_event_logs
+      )
+    }
+  }
+
+  @Test
+  fun testDeveloperOptionsFragment_overrideAppBehaviorsIsDisplayed() {
+    launch<DeveloperOptionsTestActivity>(
+      createDeveloperOptionsTestActivityIntent(internalProfileId)
+    ).use {
+      testCoroutineDispatchers.runCurrent()
+      scrollToPosition(position = 2)
+      verifyItemDisplayedOnDeveloperOptionsListItem(
+        itemPosition = 2,
+        targetView = R.id.override_app_behaviors_text_view
+      )
+      verifyTextOnDeveloperOptionsListItemAtPosition(
+        itemPosition = 2,
+        targetViewId = R.id.show_all_hints_solution_text_view,
+        stringIdToMatch = R.string.developer_options_show_all_hints_solution
+      )
+      verifyTextOnDeveloperOptionsListItemAtPosition(
+        itemPosition = 2,
+        targetViewId = R.id.force_network_type_text_view,
+        stringIdToMatch = R.string.developer_options_force_network_type
+      )
+      verifyTextOnDeveloperOptionsListItemAtPosition(
+        itemPosition = 2,
+        targetViewId = R.id.force_crash_text_view,
+        stringIdToMatch = R.string.developer_options_force_crash
+      )
+    }
+  }
+
+  @Test
+  fun testDeveloperOptionsFragment_changeConfig_overrideAppBehaviorsIsDisplayed() {
```

Comment:
`changeConfig` should be `land` since we're testing for landscape mode specifically. Ditto for everywhere else in this file -- some tests have `land` and some have `changeConfig`.

Reply:
Actually, `land` is just an alternative for `configChange`. I used this to fit the test name in the 100 char limit. This was suggested in one of my earlier PRs https://github.com/oppia/oppia-android/pull/3445#discussion_r666338923. I think there should not be any issue with this.

Reply:
I think we can do with `configChange` itself. You can think of it in this way, is OverrideAppBehaviors is displayed when we change the configuration. Does this make sense?

Reply:
Yes but like I said in the meeting, `configChange` applies to the cases where we're deliberately changing the config to check if the state persists and `land` applies to cases where we're checking in the landscape mode from the beginning. Does that help? 

Also, this applies to all the rest test cases in this file as well.

Reply:
Okay. I have renamed all the relevant tests accordingly.